### DISCUSSION TITLE
我想要使用taro/cli的taro create命令创建一个hook函数式组件页面

### DISCUSSION THREAD

**Discussion by YueHua46** [NervJS/taro#12933] [Q & A]
我想要使用taro/cli的taro create命令创建一个hook函数式组件页面，我该如何去做？
我现在暂时是在@tarojs\cli\templates\default\src\pages\index\index.jsx里把它改成了函数式写法，这样我每次create页面组件时，都是使用函数式组件写法了，这样做合适吗？想问有没有什么其他方式？

**Comment by TheKonka** [comment]
现在 taro create 命令创建的页面已经是函数组件
